src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,30 +1,64 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+import { Completed, Filters } from '../../types/Filters';
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+interface Props {
+  filterOptions: Filters;
+  onFilterChang: (key: keyof Filters, value: string | Completed) => void;
```

Comment:
There is a typo in the prop name `onFilterChang`. It should be `onFilterChange` to match the expected prop name in the parent component.